### PR TITLE
Update font API & merge upper and lower case words when stemming

### PR DESCRIPTION
**Pull request by pendragon-**
Using PIL imports in the examples, just like the main library file for consistency & compatibility.
The bigger change is moving the plural word merges so as to use the lower case words to check for existing words. This results in 'Lukes' being merged together with 'LUKE', for example.

Also fixes the following issues:
- https://github.com/amueller/word_cloud/issues/96
- https://github.com/amueller/word_cloud/issues/98

### CODE REVIEW

**Comment by pendragon-** [comment]
I am not sure what that I broke anything there:
> ImportError: libgfortran.so.1: cannot open shared object file: No such file or directory
I'd appreciate any pointers on how to fix this.

File: wordcloud/wordcloud.py [hunk 1]
```diff
@@ -361,31 +359,33 @@ def process_text(self, text):
                 continue
 
             # Look in lowercase dict.
-            if word_lower in d:
+            try:
```
Comment (amueller) [inline wordcloud/wordcloud.py]: no strong opinion on that but why not...

**Comment by amueller** [comment]
I triggered the test again, it seemed unrelated. Other than that, looks good to me. Thanks!

**Comment by amueller** [comment]
It is this issue: https://github.com/conda/conda/issues/444

I just pushed a fix to master. If master builds successfully (maybe I should have done a PR ^^) can you rebase?

**Comment by amueller** [comment]
seems to work. please rebase and let me know if the tests pass.

**Comment by pendragon-** [comment]
All done & green.

**Comment by amueller** [comment]
thanks :)

**Comment by stonebig** [comment]
could we have a small 1.2.1 to get the fix on Pypi ?

**Comment by amueller** [comment]
should be up now @stonebig

**Comment by stonebig** [comment]
Thanks, Andreas !

**Pull request merged**